WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -72,11 +76,24 @@ struct PointOfSaleDashboardView: View {
     }
 }
 
+struct FloatingControlAreaSizeKey: EnvironmentKey {
```

Comment:
Added support for environment variable for easier passing of the floating control area size